xmtp_mls/src/builder.rs

```diff
@@ -179,6 +179,27 @@ mod tests {
         assert!(!client.installation_public_key().is_empty());
     }
 
+    #[tokio::test]
+    async fn legacy_keys() {
```

Comment:
There is a quirk with this test case. It uses the legacy key the first time it is run, but after it has been run once on the local DB you will get a different result in calls to `get_inbox_ids` for that account address. That's because the first test run will register `0x0bD00B21aF9a2D538103c3AAf95Cb507f8AF1B28` with the network. If the address is already registered, the IdentityStrategy won't use the legacy key and will prompt for a wallet signature.

We should probably replace this with a dynamically generated legacy key so that we don't get confusing results in local testing where you might not drop the DB after each test run.